lib/routes/utils/password_check.js

```diff
@@ -35,26 +43,29 @@ module.exports = function (log, config, Password, customs, db) {
             return match
           }
 
-          return customs.flag(clientAddress, emailRecord)
-            .then(
-              function (result) {
-                if (result.lockout) {
-                  log.info({
-                    op: 'account.lock',
-                    email: emailRecord.email,
-                    uid: emailRecord.uid.toString('hex')
-                  })
-                  if (config.lockoutEnabled) {
-                    return db.lockAccount(emailRecord)
-                  }
+          return customs.flag(clientAddress, {
+            email: emailRecord.email,
+            errno: error.ERRNO.INCORRECT_PASSWORD
+          })
+          .then(
+            function (result) {
+              if (result.lockout) {
+                log.info({
+                  op: 'account.lock',
```

Comment:
Would it makes sense to log a reason for lockout? In this case to many failed password checks.